doc/tutorials/acrn_on_qemu.rst

```diff
@@ -267,7 +267,7 @@ Bring Up User VM (L2 Guest)
       mem_size=1024M
       acrn-dm -A -m $mem_size -s 0:0,hostbridge \
       -s 3,virtio-blk,~/UserVM.img \
-      -s 4,virtio-net,tap0 \
+      -s 4,virtio-net,tap=0 \
```

Comment:
```suggestion
      -s 4,virtio-net,tap=tap0 \
```

Reply:
As previous.
Could we use tap=LaaG to show the virtio-net format?

Reply:
We could, but what about our network scripts (the `50-*` series) that are used to set-up a `tap0` device, should we remove them from our code and let the launch scripts deal with that?